🌐 Add German translation for `docs/de/docs/tutorial/body.md`

nilslindemann opened the pull request:
← `tutorial/query-params.md` (#10293)
→ `tutorial/query-params-str-validations.md` (#10304)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/tutorial/body.md

* Full translation
* In sync with current en/docs/tutorial/body.md
* Manually translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit fc8a9963db4e46a1090902e8e2d239947a0cd13a at: https://f28a165e.fastapitiangolo.pages.dev

Commit:
Make sentence shorter

tiangolo commented:
📝 Docs preview for commit c1afd5766cff8c6948f58eccbedba50bba85a67a at: https://b36dd278.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit e18009d3210f2771afe89fcf45d6543651a7039b at: https://55d6c261.fastapitiangolo.pages.dev

Commit:
Standardwert -> Defaultwert

Passt besser. Steht auch bei Leo als erste Übersetzung

tiangolo commented:
📝 Docs preview for commit 05260a3a6716168d0d29fbb9cd7cb1db86f1ddfc at: https://8e1651a3.fastapitiangolo.pages.dev

Commit:
Fix formatting

Translating the text changed the icon and coloring of the box

tiangolo commented:
📝 Docs preview for commit c53f17fa6600974c70d63c20565de1c6712e9044 at: https://85ad290c.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit e8a239a1bad1f091822918917107a544597b3080 at: https://cd8f5c93.fastapitiangolo.pages.dev

Commit:
Improve translation of "path operation"

tiangolo commented:
📝 Docs preview for commit ae4b4186b38720a99612a970d2b64c62a184a84a at: https://93a7503d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit d20ae5c02e99f56ce11ac2765faf0199c78fcbf5 at: https://8cdcd13b.fastapitiangolo.pages.dev

Commit:
Write `Requestbody` and `Responsebody` as one word

Also add missing dash in a few locations.

tiangolo commented:
📝 Docs preview for commit 0bfa1f8e7475ca752ae33afa0babe16a3b52c2d0 at: https://daf8ddfd.fastapitiangolo.pages.dev

Commit:
A few improvements

Inspired by reading translations by @fhabers21

tiangolo commented:
📝 Docs preview for commit ba3ff87b072d52a4670e8c99f976245ed1918974 at: https://857bedba.fastapitiangolo.pages.dev

Commit:
Fix link

tiangolo commented:
📝 Docs preview for commit 5a320e306bda7fb0096d9245c41663871344dc9a at: https://85538eec.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 960db5f43f0ba7470caac9eae18f585b2d96d222 at: https://d2d20b8b.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 258ce7e00d319b3ba4c543552fb538d58963ffaa at: https://22bad6cd.fastapitiangolo.pages.dev

Commit:
"Normal" -> "Standard"

Commit:
Sync with #10442

tiangolo commented:
📝 Docs preview for commit 9f987c0441e59b841d3529512b9d7ee810b06d37 at: https://cd2f2ffb.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit eb908e7d8e29f5ea4e35a69c2dd891d4e0e038e3 at: https://49a514c9.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit c2e7ce093b34e38c97bb96ebfd28a664ce7419de at: https://f660e02d.fastapitiangolo.pages.dev

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit 639cd3777a2f7d2ad0d8326e2583c2d72141c69f at: https://3c4dba4d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 969f48ff7d5f85b8a73e183569ab57dcb871c7ec at: https://c516c5ed.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 71366e793aa235f7a72d3f5a3f9b611fb3bddf7c at: https://93fc859e.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

tiangolo commented:
📝 Docs preview for commit e0feed21363653fcdba0e4b1fb3ae5fdba4692ae at: https://85fbde09.fastapitiangolo.pages.dev

Commit:
"Python " -> "Python-", etc.

tiangolo commented:
📝 Docs preview for commit 82577463a8aef90341cc0b908a7c3b7bf9a4628d at: https://f2b55c5a.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 9c2269aba453fbebef8d5bad443b25ff0de26816 at: https://7836b45d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit 4a814d20b7660f20fa34d2490b55b976089abc2f at: https://79c92e33.fastapitiangolo.pages.dev

Commit:
"Modell-Attribut" -> "Modellattribut"

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit e461c494d8ddf9f7fdb0d549a80b06301e704f1f at: https://a2a6ab63.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

tiangolo commented:
📝 Docs preview for commit cbe9e1c871b9435684f9136212d2edec677dde4c at: https://93089602.fastapitiangolo.pages.dev

mattkoehne commented on docs/de/docs/tutorial/body.md:
````diff
@@ -0,0 +1,213 @@
+    {!> ../../../docs_src/body/tutorial004.py!}
+    ```
+
+die Funktionsparameter werden wie folgt erkannt:
````
`die` -> `Die`

mattkoehne commented on docs/de/docs/tutorial/body.md:
```diff
@@ -0,0 +1,213 @@
+* Wenn der Parameter vom Typ eines **Pydantic-Modells** ist, wird er als Request**body** interpretiert.
+
+!!! note "Hinweis"
+    FastAPI weiß, dass der Wert von `q` nicht erforderlich ist, wegen des definierten Defaultwertes `=None`
```
`=None` -> `= None`

mattkoehne reviewed (changes requested):
thanks @nilslindemann  🍰

nilslindemann commented on docs/de/docs/tutorial/body.md:
```diff
@@ -0,0 +1,213 @@
+
+Ein **Request**body sind Daten, die vom Client zu ihrer API gesendet werden. Ein **Response**body (Deutsch: Antwortkörper) sind Daten, die Ihre API zum Client sendet.
+
+Ihre API sendet fast immer einen **Response**body. Aber Clients senden nicht unbedingt immer **Request**bodys (sondern nur Metadaten).
```
Ich hab diese Änderung in einem [anderen PR](https://github.com/tiangolo/fastapi/pull/10985/commits/049af7eb2513324c251f3ac3c6df7b58a63dd82d#diff-43ba58c42d0a363badc01ac2a14030f6e34e498c2af4c9d720396a36e60bea29) vorgeschlagen. Wahrscheinlich hätte ich das in einem eigenen PR unterbringen sollen. Dumm gelaufen.

Ich plane, das, was in dem PR nicht übernommen wird, dann in der deutschen Doku rückgängig zu machen. Das ist so einfacher für mich, sonst muss ich streng genommen alles rückgängig machen und dann eventuell nachher alles wiederherstellen.

Ich lass' das jetzt mal drin, hab aber ein Auge drauf.

Commit:
Apply fixes from @mattkoehne

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/body.md

nilslindemann commented:
Danke wieder, Matt :+1:

tiangolo commented:
📝 Docs preview for commit 189b25b78458d94e35b97fbdf7c0a45d3ab041e1 at: https://7b11ca75.fastapitiangolo.pages.dev

mattkoehne reviewed (approved):
Thank you @nilslindemann 🚀

alejsdev commented:
Thanks for your help! @nilslindemann @mattkoehne 🐱🎉
